protocol/testing/delaymsg_config/perpetual_fee_params_msg.json

```diff
@@ -56,15 +56,15 @@
         "absolute_volume_requirement": "125000000000000",
         "total_volume_share_requirement_ppm": 5000,
         "maker_volume_share_requirement_ppm": 10000,
-        "maker_fee_ppm": -90,
+        "maker_fee_ppm": -70,
```

Comment:
Makes sense. Alternatively make sense to change directory of `testing` to something like `networks` since it really is used for setting up production and test networks

Filed ticket [here](https://linear.app/dydx/issue/CORE-633/mov-files-used-by-prod-genesis-out-of-the-testing-directory)